src/Build/BuildCheck/Acquisition/BuildCheckAcquisitionModule.cs

```diff
@@ -42,18 +42,17 @@ public List<BuildAnalyzerFactory> CreateBuildAnalyzerFactories(AnalyzerAcquisiti
             assembly = Assembly.LoadFrom(analyzerAcquisitionData.AssemblyPath);
 #endif
 
-            IEnumerable<Type> analyzerTypes = assembly.GetExportedTypes().Where(t => typeof(BuildAnalyzer).IsAssignableFrom(t));
+            IList<Type> availableTypes = assembly.GetExportedTypes();
+            IList<Type> analyzerTypes = availableTypes.Where(t => typeof(BuildAnalyzer).IsAssignableFrom(t)).ToArray();
```

Comment:
```suggestion
            IList<Type> analyzerTypes = availableTypes.Where(t => t.IsInstanceOfType(typeof(BuildAnalyzer))).ToList();
```

Super nits: 1) `IsInstanceOfType` reads better here (it expresses the intent), 2) `ToList()` is prefferable as `ToArray()` needs to copy twice. In this specific case it doesn't hurt, but it's good to keep sticking to this.